### PR TITLE
fix: Special case paging for ubuntu entries

### PR DESCRIPTION
**Pull request by another-rex**
Add a special case for ubuntu entries to reduce the page size by 10x. Because Ubuntu entries are incredibly large.

### CODE REVIEW

File: gcp/api/server.py [hunk 1]
```diff
@@ -733,6 +741,12 @@ def do_query(query: osv_service_v1_pb2.Query,
         'version specified in params and purl query',
     )
 
+  # Hack to work around ubuntu having extremely large individual entries
+  if (ecosystem.startswith("Ubuntu") or
```
Comment (oliverchang) [inline gcp/api/server.py:745]: nit: single quotes for strings.

**Review by oliverchang — APPROVED** [review]

File: gcp/api/server.py (hunk 1 above)
Comment (hogo6002) [inline gcp/api/server.py:745]: I think only `linux` related packages would reach the limit. Should we only apply this to packages start with `linux`?

**Review by hogo6002 — APPROVED** [review]
Thanks Rex!

File: gcp/api/server.py (hunk 1 above)
Comment (another-rex) [reply gcp/api/server.py]: Applied to only get linux packages.

**Pull request merged**